common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -37,7 +37,11 @@ trait Logging {
    * @param message Message to write to the log
    */
   def debug(from: AnyRef, message: String)(implicit id: TransactionId = TransactionId.unknown) = {
-    emit(DebugLevel, id, from, message)
+    if (id.meta.extraLogging) {
```

Comment:
should this be applied to _all_ the logger methods not just debug perhaps by making the change in the `emit` method instead?

Reply:
The only message we do suppress when running on INFO level are the ones on debug level. 
No need to generalize in my opinion.

Reply:
is that true for warnings?

Reply:
I second this. We should apply this to all methods (possibly do it just in `emit`?)! If we decide to run on WARN or on ERROR, you still want *all* logs to appear for those special users, no?

Reply:
@rabbah  WARNINGS will be shown when running on INFO level
@markusthoemmes Do you really think running OW on a level higher than INFO is a good idea with the current logging output ?

Reply:
this still feels wrong to me - it presupposes that you're logging at least at InfoLevel - where the intent is to enable logging, period. So if the log level is off, this flag generates no additional output. Is that the intend?